devconf-ironic-and-edgy/source/index.rst

```diff
@@ -11,7 +11,7 @@ In this talk...
 * Bare Metal provisioning at the Edge
 * Ironic at the Edge: now and in the future
 
-.. Things to talk about ^
+.. Things to talk about ^ (ietingof)
```

Comment:
@dtantsur Let's figure who is talking what? Here I came up with some division, feel free to reshuffle it in any way ;)